### PR TITLE
Update Node API `data_available`

### PR DESCRIPTION
**Pull request by YazzyYaz**
Closes https://github.com/celestiaorg/docs/issues/140

### CODE REVIEW

File: docs/developers/instantiate-testnet.md [hunk 1]
```diff
@@ -7,106 +7,15 @@ to test out new features to build as a core developer.
 
 ## Hardware Requirements
 
-The following hardware minimum requirements are recommended for running the
```
Comment (adlerjohn) [inline docs/developers/instantiate-testnet.md:10]: One PR per issue. Do this or that in a separate PR.

File: docs/developers/node-tutorial.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ You can follow the tutorial for setting up the dependencies [here](./environment
 
 ## Celestia Node
 
-## Install Celestia Node
+### Install Celestia Node
```
Comment (adlerjohn) [inline docs/developers/node-tutorial.md:17]: ditto

**Comment by adlerjohn** [comment]
Converting to draft until unblocked by https://github.com/celestiaorg/celestia-node/pull/802 being finalized.

File: docs/developers/node-api.md [hunk 3]
```diff
@@ -2119,6 +2119,25 @@ Returns
...
+curl -X GET http://<ip>:26658/data_available/25000
+```
+
+Response
+
+```json
+{
+   "height":25000,
+   "available":true
+}
```
Comment (renaynay) [inline docs/developers/node-api.md:2138]:
```suggestion
{"available":true,"probability_of_availability":"0.9899774042423815"}
```

**Review by renaynay — CHANGES_REQUESTED** [review]

File: docs/developers/node-api.md [hunk 4]
```diff
@@ -2119,6 +2119,25 @@ Returns
 }
 ```
 
+#### GET /data_available/{height}
+
+Returns whether data is available at a specific block height.
```
Comment (renaynay) [inline docs/developers/node-api.md:2124]:
```suggestion
Returns whether data is available at a specific block height and the probability that it is available based on the amount of samples collected.
```

**Review by renaynay — APPROVED** [review]

**Pull request merged**